portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -156,6 +156,21 @@
 	<td></td>
 </tr>
 <!--DATE-->
+<tr>
+	<td>DATE_CALENDAR</td>
+	<td>//button[@aria-label='Choose date']</td>
```

Comment:
@lima-rafa, let's use a CSS class instead of `aria-label,` for example `date-picker-dropdown-toggle`. The `aria-label` is something related to Accessibility, therefore I understand that using the suggested CSS class will improve the path comprehension and also remove any association with Accessibility.